collaboration/sig-core/proposals/image-name-consolidation.md

```diff
@@ -0,0 +1,98 @@
+# Container Image Name Consolidation
+
+## Introduction
+In order to run kyma on a kubernetes cluster a lot of different container images are generated. These images are referenced in templates used by the helm charts that are installed by the `kyma-installer`
+
+## Problem
+Currently there is not always a clear relationship between an images name and the source code that resulted in the image.
+
+As a result:
+- It's not easily possible to find the source code for a particular image
+- It's not easily possible to find the latest image that was built using the source code
+
+## Goal
+
+Propose a name pattern for container images that helps users quickly find images from source and vice versa.
+
+##Conventions
+**cmd** its is a golang convention to put source code for the main executable in a folder called `cmd`. A subfolder should exist for every executable that shall be created from the project.
+e.g.
+```shell
+component-name
+└- cmd
+   ├- executable_a
+   |  └- main.go
+   └- executable_b
+      └- main.go
+```
+
+**tests** Tests are sorted into the `test`-folder following this proposal [https://github.com/kyma-project/community/blob/master/collaboration/sig-core/proposals/test-folder-consolidation.md]
+
+
+## Proposal
+
+The path to each binary is unique. Significant information contained in the path of a binary should be used as the base for the image name that contains the binary.
+
+### Components:
+
+#### Pattern:
+
+| path | image name | rule |
+| ---- | ---------- | ---- |
+| `components/<component-name>/cmd/<component-binary>` | `<component-name>-<component-binary>` | applies if multiple images can be built using the component-folder |
+| `components/<component-name>/cmd/<component-binary>` | `<component-name>` | applies if only a single images can be built using the component-folder |
```

Comment:
just add info that `cmd` is not required and if someone has only one `main.go` on root directory then the same pattern applies :)